content/features/featuresDeepDive/lights/lights_introduction.md

```diff
@@ -61,6 +61,20 @@ that the full effect is achieved.
 const light = new BABYLON.HemisphericLight("HemiLight", new BABYLON.Vector3(0, 1, 0), scene);
 ```
 
+### The Rectangular Area Light
+
+A rectangular area light is defined by its position, width, and height. It emits light from the resulting surface towards the -Z direction. Even though the RectAreaLight class itself does not have a direction component, it can be attached to a transform node to be rotated and moved around.
+
+_A simple use of a rectangular area light_

```

Comment:
Are we missing an image here? The "simple use of a rectangular area light" seems to imply there should be an image.